packages/components/focus-trap/src/focus-trap.vue

```diff
@@ -126,16 +131,18 @@ export default defineComponent({
         const isPrevFocusContained = trapContainer.contains(prevFocusedElement)
         if (!isPrevFocusContained) {
           const mountEvent = new Event(FOCUS_ON_MOUNT, FOCUS_ON_MOUNT_OPTS)
-          on(trapContainer, FOCUS_ON_MOUNT, focusOnMount)
+          trapContainer.addEventListener(FOCUS_ON_MOUNT, focusOnMount)
           trapContainer.dispatchEvent(mountEvent)
           if (!mountEvent.defaultPrevented) {
-            focusFirstDescendant(
-              obtainAllFocusableElements(trapContainer),
-              true
-            )
-            if (document.activeElement === prevFocusedElement) {
-              tryFocus(trapContainer)
-            }
+            nextTick(() => {
```

Comment:
Using await nextTick here will look more beautiful！

Reply:
This method is not really an async method actually, so I think we should keep the callback style here since there were no nested callbacks.

Reply:
OK